Extension/tools/OptionsSchema.json

```diff
@@ -561,7 +561,7 @@
             "%c_cpp.debuggers.cppvsdbg.console.newExternalWindow.description%"
           ],
           "description": "%c_cpp.debuggers.cppvsdbg.console.description%",
-          "default": "internalConsole"
+          "default": "externalTerminal"
```

Comment:
Is this correct?

Reply:
No. I believe `default` should indicate the value it will be if it is not set. 

So the default value of console is `integratedTerminal`.

Reply:
but package.json has it as "internalConsole" -- you mean the value in package.json is incorrect?

Reply:
`internalConsole` is correct and `package.json` properties for the debugger are generated from `OptionsSchema.json`. So the value before this change is correct.